core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -115,6 +117,8 @@
    * Adds interceptors that will be called before the channel performs its real work. This is
    * functionally equivalent to using {@link ClientInterceptors#intercept(Channel,
    * ClientInterceptor...)}, but while still having access to the original {@code ManagedChannel}.
+   * Interceptors run in the reverse order in which they are added, just as with consecutive calls
```

Comment:
Optional: ClientCall.Listener events happen in the same order as interceptors though.  It's only reverse going down, which may be not obvious.

Reply:
I'm not sure how to discuss that without it being even more confusing. To me, just the fact that we mention this should clue most users in, even if they aren't 100% certain what it implies when they read it first.